tests/nip46/validator-unit.test.ts

```
@@ -0,0 +1,748 @@
+         NIP46SecurityValidator.validatePrivateKeySecure("", "test key");
+       }).toThrow("test key is required and cannot be empty");
+
+       // Test in production mode
+       process.env.NODE_ENV = 'production';
+       
+       expect(() => {
+         NIP46SecurityValidator.validatePrivateKeySecure("", "test key");
+       }).toThrow("Invalid key format");
+
+       // Reset to original state
+       delete process.env.NODE_ENV;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)